fdbserver/masterserver.actor.cpp

```diff
@@ -2002,7 +2021,7 @@ ACTOR Future<Void> masterServer(MasterInterface mi,
 		TEST(err.code() == error_code_commit_proxy_failed); // Master: terminated because of a commit proxy failure
 		TEST(err.code() == error_code_grv_proxy_failed); // Master: terminated because of a GRV proxy failure
 		TEST(err.code() == error_code_master_resolver_failed); // Master: terminated because of a resolver failure
-		TEST(err.code() == error_code_master_backup_worker_failed); // Master: terminated because of a backup worker failure
+		TEST(err.code() == error_code_master_backup_worker_failed); // Master: terminated because of a BW failure
```

Comment:
I think the original comment is better/clearer. I guess you want something like `backup worker (BW)`?

Reply:
I reverted this. `clang-format` was formatting it so I had this line smaller just that I don't get coverage failure.